app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/max_height/AddMaxPhysicalHeight.kt

```diff
@@ -47,7 +48,13 @@ class AddMaxPhysicalHeight(
         and !maxheight:physical
         and access !~ private|no
         and vehicle !~ private|no
+        and motorroad !~ yes
+        and motorway !~ yes
+        and expressway !~ yes
     """.toElementFilterExpression() }
+    // explicitly removed motorway-style ways (highway=motorway*, motorroad/motorway=yes,
+    // expressway=yes) from ever being matched as measuring heights on these ways can be
```

Comment:
```suggestion
    // expressway=yes) from ever being matched as measuring heights on these ways likely is```

can be on any road

Reply:
The comment is broken in your suggestion lines, isn't it? I'd have commited it otherwise :)

Reply:
should be fixed now